static/app/views/performance/landing/widgets/components/accordion.tsx

```diff
@@ -0,0 +1,91 @@
+import {ReactNode} from 'react';
+import styled from '@emotion/styled';
+
+import DropdownButton from 'sentry/components/dropdownButton';
+import space from 'sentry/styles/space';
+
+type Props = {
+  content: ReactNode;
+  items: (() => ReactNode)[];
+  selectedIndex: number;
+  setSelectedIndex: (index: number) => void;
+};
+
+export default function Accordion({
+  content,
+  items,
+  selectedIndex,
+  setSelectedIndex,
+}: Props) {
+  return (
+    <AccordionContainer>
+      {items.map((item, index) => (
+        <AccordionItem
+          isSelected={index === selectedIndex}
+          currentIndex={index}
+          key={index}
+          content={content}
```

Comment:
You can ignore this comment outright _but_ I think most accordion components expect to get a list of items, in which each item specifies both its header and its content. That's usually a clearer API than passing a list of item headers and one body since _usually_ the accordion decides on how to toggle what's visible, and it's a lot easier to do that if it has all the content upfront.

I don't think your `Accordion` needs to work that way, but if it did, it would be easier to make it generic! Just food for thought

Reply:
yeah agreed, it makes more sense to have `content` for each `header`. I might need to rework the widget to make sure it works that way. Will follow up with a refactor PR 👍